html/frontend.html

```diff
@@ -26,65 +26,131 @@
 		<!-- toolbar -->
```

Comment:
I never touched the file but you are right that the pre-commit hook messed with it. I assume because of changes that came with the merging in b10bdc4. Should be reverted now.